assets/js/race.js

```diff
@@ -168,5 +168,55 @@ function postSignup() {
 
 }
 
+/*
+ * This function is called when clicking 'save'.
+ * It will store the individual information into cookie.
+ */
+function cacheIndividual() {
+    var data =[];
+    $(".individual-form[class!='individual-form hidden']").each(function() {
+        var name = $("input[name='name']", this).val();
+        var gender = $("select[name='gender']", this).val();
+        var id_card = $("input[name='id_card']", this).val();
+        var race = $("select[name='race']", this).val();
+        var accommodation = $("input[name='accommodation']", this).val() == "on" ? "1" : "0";
```

Comment:
I am not sure but I wonder how do we set the value for the input[name='accommodation'], If you need to set it as on or off, we should just save it as it is.